cfgov/alerts/post_sqs_messages.py

```diff
@@ -1,9 +1,11 @@
+from __future__ import unicode_literals
+
 import argparse
 import boto3
-import json
 import logging
-import github3
-import requests
+
+from github_alert import GithubAlert
+from mattermost_alert import MattermostAlert
```

Comment:
Could these imports be absolute instead?

Reply:
Why wouldn't that work? How is this script invoked? At a minimum wouldn't it be nice to make these imports explicitly relative (e.g. `from .github_alert import GitHubAlert`)?

Reply:
It's invoked directly, `python cfgov/alerts/post_sqs_messages.py`, so the `PYTHONPATH` doesn't include the `alerts` package.

I agree, I'd like to see more explicit imports. I have a soft preference for requiring the `PYTHONPATH` to be set explicitly to include `cfgov/` and then absolute-importing them with `alerts.github_alert`.

Reply:
If `alerts` was added as a top-level app to `settings.INSTALLED_APPS`, could this be converted to a management command?

Reply:
But we would then have to have a full installation of Django and all our requirements in the Jenkins workspaces where this (and the New Relic) script runs. That seems like massive overkill.

Reply:
Updating to be absolute